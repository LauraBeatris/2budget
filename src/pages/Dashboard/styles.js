import styled from "styled-components";

export const DashboardContainer = styled.div`
  text-align: center;
  margin-top: 3rem;
  label {
    font-size: 2rem;
    color: rgb(77, 81, 94);
  }
`;
