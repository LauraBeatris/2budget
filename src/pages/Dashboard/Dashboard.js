import React, { Component } from "react";
import Helmet from "react-helmet";
import ExpenseList from "../../components/ExpenseList/ExpenseList";
import ExpenseListFilter from "../../components/ExpenseListFilter/ExpenseListFilter";
import { DashboardContainer } from "./styles";

export default class Dashboard extends Component {
  render() {
    return (
      <div>
        <Helmet>
          <title> 2Budget | Dashboard </title>
        </Helmet>
        <DashboardContainer>
          <label>Filters</label>
          <ExpenseListFilter />
          <ExpenseList />
        </DashboardContainer>
      </div>
    );
  }
}
