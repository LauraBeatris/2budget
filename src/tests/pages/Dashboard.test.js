import { shallow } from "enzyme";
import React from "react";
import Dashboard from "../../pages/Dashboard/Dashboard";

test("should render Dashboard correctly", () => {
  const wrapper = shallow(<Dashboard />);
  expect(wrapper).toMatchSnapshot();
});
