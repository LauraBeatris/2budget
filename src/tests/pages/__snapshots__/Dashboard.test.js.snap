// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard correctly 1`] = `
<div>
  <Connect(Header) />
  <.sc-EHOje>
    <label>
      Filters
    </label>
    <Connect(ExpenseListFilter) />
    <Connect(ExpenseList) />
  </.sc-EHOje>
</div>
`;
